src/notebooks/outputs/errorRendererComms.node.ts

```diff
@@ -68,13 +68,14 @@ export class ErrorRendererCommunicationHandler implements IExtensionSyncActivati
                 const lineNumber = parseInt(lineMatch[1], 10);
                 selection = new Range(new Position(lineNumber, 0), new Position(lineNumber, 0));
             }
+            uri = uri.with({ query: '' });
```

Comment:
This fixes the opening a new file. With the query on the uri, the check to match below fails.